workflow/defaults/fv3_gfs.yaml

```diff
@@ -116,7 +116,6 @@ fv3_gfs_defaults: &fv3_gfs_defaults
            hord_xx_nh_nonmono: 5
            vtdm4_nh_nonmono: 0.02
            nord: 2
-           dddmp: 0.1
```

Comment:
Does it not work if this is set? My understanding is these values should be getting overridden by what is in the case file, and if not I should dig in and figure it out. If the defaults are removed, workflow might fail if dddmp is *not* set.

Reply:
Replied on another line in response to this.